CHANGELOG.md

```diff
@@ -46,6 +46,34 @@ All notable changes for each version of this project will be documented in this
     - `multi` mode - select/deselect all dates between the last selected/deselected and the one clicked while holding `Shift`.
     - `range` mode - extend/shorten the range from the last selected date to the one clicked while holding `Shift`.
 
+### Theme Changes
+- **Breaking Changes** - The `palette` function no longer provides `info`, `success`, `warn` and `error` colors. Therefore you have to pass custom values for them if you need to use these colors. You can also use the values for `info`, `success`, `warn` and `error` colors from our predefined color palettes.
+
+    - Code example:
+
+    ```scss
+    // Mandatory colors
+    $primary-color: #2a38b7;
+    $secondary-color: #f96a88;
+    $surface-color: #e1ebe4;
+    // Additional colors
+    $error-color: color($light-fluent-palette, 'error');
```

Comment:
I wonder if it's possible to migrate this type of calls:

```scss
$app-palette: palette($primary: $primary, $secondary: $secondary, $grays: $grays);
@include dark-theme(
    $palette: $app-palette,
    $exclude: map-keys(map-remove($components, $include...),)
);
```

to this:

```scss
$app-palette: palette($primary: $primary,
  $secondary: $secondary,
  $grays: $grays,
  // auto-migrated to material dark based on dark-theme preset call
  $error: color($dark-material-palette, 'error'),
  $warn: color($dark-material-palette, 'warn'),
  $info: color($dark-material-palette, 'info'),
  $success: color($dark-material-palette, 'success')
);
@include dark-theme(
    $palette: $app-palette,
    $exclude: map-keys(map-remove($components, $include...),)
);
```

Reply:
Just to mention that the same thing happened with the `greys` in one of the previous updates and wasn't auto-migrated, so we have to think about how to handle these cases automatically.

Reply:
@SisIvanova I'm not sure if you should answer this question. Your PR is ready for merge, but since there's a hanging discussion here, I would like to get an answer prior to merging it, so it doesn't get lost.

Reply:
@kdinev Yes, we are planning to provide fallback values for these colors and it should be done soon.

Reply:
@SisIvanova So, if there's going to be a default fallback, then a migration would not be needed, I guess?

Reply:
@kdinev Yes, no migration is needed.